paper.tex

```diff
@@ -543,7 +540,7 @@ \subsection{Datasets} % 1/4 of a page (half a col)
 partitioned into 125 chunks.
 
 Processing large images is only considered to be part of the Big Data problem 
-in neuroscience. The other problem is processing large MRI datasets, 
+in neuroscience. The other being the processing large MRI datasets, 
```

Comment:
is a word missing here?

Reply:
neuroscience is the last noun, but 'problem' is the only noun that can have an 'other'. if it's not clear, i can add 'problem' back

Reply:
ok then